src/observers/strategies/cpu_contention.hpp

```diff
@@ -50,26 +50,22 @@ class CpuContentionStrategy : public RevocationStrategy {
       const SerenityConfig& _config,
       const lambda::function<usage::GetterFunction>& _cpuUsageGetFunction)
       : RevocationStrategy(Tag(QOS_CONTROLLER, "CpuContentionStrategy")),
-        cpuUsageGetFunction(_cpuUsageGetFunction),
-        cooldownCounter(None()),
-        estimatorDisabled(false) {
+        getCpuUsage(_cpuUsageGetFunction) {
     SerenityConfig config = CpuContentionStrategyConfig(_config);
-    this->cfgCooldownTime = config.getU64(strategy::CONTENTION_COOLDOWN);
-    this->cfgDefaultSeverity = config.getD(strategy::DEFAULT_CPU_SEVERITY);
   }
 
-  RevocationStrategyFunction decide;
+  Try<QoSCorrections> decide(ExecutorAgeFilter* ageFilter,
```

Comment:
I think developer 'translates' only once. When viewing the code couple of times you not need to know the entire, detailed info about the parameters, just to know what it does and why. 

I'm not convinced, but feel free to do as you proposed (: